themes/default/content/testimonials/_index.md

```diff
@@ -0,0 +1,161 @@
+      source: email
+    -
+      message: "Hi Joe,\n\nLooks good. Coming from 5 years of terraform I find it easier for me and the team. We are going to ramp our efforts and try to build our entire infra on it.\n\nBest,\nAlon"
+      author: Alon
+      link: https://docs.google.com/presentation/d/11IncvcImn6ZFLu8Vhla1UgL-ugtLFPj4v321AijE5Tc/edit#slide=id.g13285f2d057_1_422
+      source: email
+    -
+      message: "Hello Joe,\n\nI tested Pulumi and followed one of the workshops offered by Lee and I was amazed at the possibilities compared to TF. We are starting to hear about Pulumi with our clients, so I can see myself delivering production-ready IaC with Pulumi in the near future. Something quite special you all have built."
+      author: Jose
+      link: https://docs.google.com/presentation/d/11IncvcImn6ZFLu8Vhla1UgL-ugtLFPj4v321AijE5Tc/edit#slide=id.g13285f2d057_1_422
+      source: email
+    -
+      message: I am using pulumi a lot these days just to look at an alternative to cdk8s. Very nice product.
+      author: Ludovic
+      link: https://docs.google.com/presentation/d/10bEPeIYTcxEO3qJLQPOz9fyIsZUfJT10qRIENAYwtlk/edit#slide=id.g1275cf56285_15_897
+      source: email
+    -
+      message: "Hi Joe,\n\nI have been long time user of AWS CDK and now I am trying Pulumi.\n\nLooks great so far and exceeds my expectations"
+      author: Nemanja
+      link: https://docs.google.com/presentation/d/10bEPeIYTcxEO3qJLQPOz9fyIsZUfJT10qRIENAYwtlk/edit#slide=id.g1275cf56285_15_897
+      source: email
+    -
+      message: Pulumi is awesome. I love it. Thanks for making awesome stuff.
+      author: Yasith
+      link: https://docs.google.com/presentation/d/1jMinKtVH5HgvSZgXmuKGegQyLsN0782HFS3NaMyjMic/edit#slide=id.g12308ba8081_1_455
+      source: email
+    -
+      message: "Hi Joe,\n\nFeedback? I will be honest.\n\nI love it!\n\nI am a consulting software engineer and my primary language is C#. I enjoyed a lot while creating resources with the language I prefer and doing that without any extra knowledge!\n\nI will be your tech evangelist in the companies I am working with.\n\nThanks for this great product.\n\n"
+      author: Ayberk
+      link: https://docs.google.com/presentation/d/1-5T3r9oF4V6vXMTLgufMdIx3_Ffl-zNOAsbb5Jg20gQ/edit#slide=id.g118d1455e14_2_4100
+      source: email
+    -
+      message: "Let me tell you Joe. I''m ******* impressed. Been using serverless framework for 5 years. Holy ****. You''ve got something here"
+      author: Andrew
+      link: https://docs.google.com/presentation/d/1fgk8314WyfZdD4dKJkdax3FA0MLi4stE5bCAhuP-wwE/edit#slide=id.ge4534e2623_3_567
+      source: email
+    -
+      message: "Hi Joe,\n\nI feel your product is awesome and the code examples are great and what made me like Pulumi is your constant support and documentation related emails. Keep doing what you are doing and I hope it will be the best IAC tool for the future."
```

Comment:
```suggestion
      message: "I feel your product is awesome and the code examples are great and what made me like Pulumi is your constant support and documentation related emails. Keep doing what you are doing and I hope it will be the best IAC tool for the future."
```